### ISSUE TITLE
Crash when include:: a wrong filetype

### ISSUE THREAD

**Issue by seriouslyEnter** [asciidocfx/AsciidocFX#252]
When I accidently "include::" a .pdf instead of .adoc AsciidocFX crashes.

**Comment by rahmanusta** [comment]
You are right, there can be two reason, one reason is file size other reason is if webview try to render it. I suppose webview will not try to render a PDF, so I assume it is about file size. What was file size that you try?

Thanks

**Comment by seriouslyEnter** [comment]
I tryied some pdf filesizes now. It crashes somewhere between 2 - 2,4MB

**Comment by rahmanusta** [comment]
It is normal,  but we should warn users for that
